feature2/f2-one.txt

```diff
@@ -0,0 +1,3 @@
+This is feature branch-2 and fine file in feature folder
```

Comment:
Hi It is really good that you added this line of code , great learning

Reply:
Thanks for you time :-)